todo

```diff
@@ -0,0 +1,5 @@
+ens to register:
+
+wrap://ens/logger.polytest.eth -> wrap://ens/logger.core.polywrap.eth
```

Comment:
What do you mean by `logger.core`?

Reply:
I mean to literally register that under the subdomain of `core.polywrap.eth` - The reason is that that's how we named it in core https://github.com/polywrap/monorepo/blob/prealpha/packages/js/core/src/interfaces/core-interface-uris.ts#L5

Reply:
Why do we need to put it under the core though?

Reply:
Not strong opinion on this, I guess it makes sense to have the subdomain core? Let's discuss this next week